.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+fail_fast: true
```

Comment:
pre-commit is a great tool for managing repos, TLDR it will install a git hook so that every time you commit code it will run a series of checks based on this configuration. This includes things like making sure no ssh keys / AWS credentials were accidently committed with code (very common mistake when using notebooks etc), as well as file formatting.

to use:
`pip install pre-commit`
`pre-commit install`

and after that will automatically run on every commit.